ui/tests/acceptance/secrets/backend/totp/key-test.js

```diff
@@ -13,19 +13,15 @@ import { mountBackend } from 'vault/tests/helpers/components/mount-backend-form-
 import { v4 as uuidv4 } from 'uuid';
 import sinon from 'sinon';
 
-const SELECTORS = {
-  menuItem: (item) => `[data-test-popup-menu="${item}"]`,
-};
-
 module('Acceptance | totp key backend', function (hooks) {
   setupApplicationTest(hooks);
 
-  const createVaultKey = async (keyName, issuer, accountName) => {
+  const createVaultKey = async (keyName, issuer, accountName, exported = true) => {
     await fillIn(GENERAL.inputByAttr('name'), keyName);
     await fillIn(GENERAL.inputByAttr('issuer'), issuer);
     await fillIn(GENERAL.inputByAttr('accountName'), accountName);
+    if (!exported) await click(GENERAL.ttl.toggle('toggle-exported'));
```

Comment:
I'm not sure if we have a policy about this, but it's usually best to avoid one line if statements. 

```suggestion
    if (!exported) {
      await click(GENERAL.ttl.toggle('toggle-exported'));
    }
```